pkg/controller/druid/handler.go

```diff
@@ -328,6 +339,44 @@ func deployDruidCluster(sdk client.Client, m *v1alpha1.Druid) error {
 	return nil
 }
 
+func checkCrashStatus(sdk client.Client, m *v1alpha1.Druid) {
+	podList := podList()
+	listOpts := []client.ListOption{
+		client.InNamespace(m.Namespace),
+	}
```

Comment:
would this actually list all the pods in the namespace ?
There could be other pods in the namespace besides those managed by the operator itself. 
Is there a way to filter the pods which are managed by the druid-operator vs those not-managed by druid operator ?

Reply:
oh i missed this will filter on label ```app=druid```